_posts/2024-01-10-spanning-tree-topology.md

```diff
@@ -0,0 +1,16 @@
+---
+title: February 5th ESI Network Downtime
+state: active
+severity: 0
+
+---
+
+Hello ESI Users,
+
+Downtime is planned for February 5th, 2024 for parts of the MOC Alliance network. This will affect ESI and a few other switches. We will be deploying a new spanning tree topology which will require restarting many of the production switches. Please make sure all work on the ESI nodes is ready for network downtime on the 5th. We will let you know in this 
+channel once the work is done and the network is working again.
```

Comment:
what does "this channel" refer to?

Reply:
me moving to fast :) will update with status page.

Reply:
Downtime is planned for February 5th, 2024 starting at 10am EST for parts of the MOC Alliance network. This will affect ESI and projects which rely on it.  Please make sure all work on the ESI nodes is ready for network downtime on the 5th. We will let you know in this channel once the work is done and the network is working again. The downtime is to deploy a new spanning tree topology, which will require restarting many of the production switches.

Reply:
updated to include @msdisme changes and remove the hello, so the main page will have more than just "Hello ESI users,"